### PR TITLE
doc: replace banner with the TEAM plan info and some other minor change

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

File: pages/pricing/index.vue [hunk 1]
```diff
@@ -437,16 +499,16 @@ export default defineComponent({
   methods: {
     onButtonClick(plan: Plan) {
       if (plan.type === PlanType.TEAM) {
-        window.open(
-          "https://hub.bytebase.com",
-          "__blank"
-        );
+        window.open("https://hub.bytebase.com", "__blank");
```
Comment (ecmadao) [inline pages/pricing/index.vue:502]: Do we need to update the URL to hub.bytebase.com/pricing?
Pros: users can create account and purchase directly.
Cons: that page is quite similar with this page, may make users a little confused.

**Review by ecmadao — COMMENTED** [review]
Left a comment. LGTM otherwise

File: pages/pricing/index.vue (hunk 1 above)
Comment (tianzhou) [reply pages/pricing/index.vue:502]: I am thinking we can add a plan query parameter to the hub.bytebase.com/pricing 

hub.bytebase.com/pricing?plan=team

so if the route get the query parameter, we popup the payment dialog directly

wdyt?

Comment (ecmadao) [reply pages/pricing/index.vue:502]: sounds good to me, will update the hub code later.

Comment (tianzhou) [reply pages/pricing/index.vue:502]: SG, we can add the count and org query parameter as well to prefill the field

Comment (tianzhou) [reply pages/pricing/index.vue]: Done

**Comment by tianzhou** [comment]
PTAL

**Review by ecmadao — APPROVED** [review]
LGTM

**Pull request merged**